stack.yaml

```diff
@@ -111,6 +111,11 @@ packages:
     commit: 2d261bb971bada1893753b503452d9e6e217bc4a
   extra-dep: true
 
+- location:
+    git: https://github.com/neongreen/servant-generic.git
```

Comment:
Maybe move it to `serokell/` to make our lives potentially easier if you are hit by a bus?

Reply:
done